src/main/java/org/jenkins/tools/test/hook/SkipUIHelperPlugins.java

```diff
@@ -37,16 +35,10 @@ public List<String> transformedPlugins() {
         return Collections.unmodifiableList(allBundlePlugins);
     }
 
-    /**
-     * The plugin was identified as something that should be skipped. Create a {@link
-     * TestExecutionResult} preventing forward movement. Also, indicates that we should skip the
-     * checkout completely.
-     */
     @Override
     public Map<String, Object> action(Map<String, Object> moreInfo) {
         LOGGER.log(Level.WARNING, "Plugin unsupported at this time, skipping");
-        moreInfo.put("executionResult", new TestExecutionResult(new ExecutedTestNamesDetails()));
-        moreInfo.put("runCheckout", false);
+        moreInfo.put("skipPlugin", true);
```

Comment:
Since we've deleted `TestExecutionResult` we now use the `skipPlugin` property as a marker to indicate that this plugin should be skipped. This should probably be revisisted in the future but we didn't want to go down that rabbit hole in this PR.